src/Microsoft.DotNet.Build.Tasks/PackageFiles/GenerateAssemblyInfo.Sdk.targets

```diff
@@ -0,0 +1,66 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- BuildTools never generated this attribute before, so don't let the SDK generate it. -->
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+
+    <!--
+    When using the Microsoft.NET.Sdk, Company and Product are defaulted to $(AssemblyName)
+    in Microsoft.NET.Sdk.BeforeCommon.targets, which means we would have to default them in
+    a props file. However, the versioning.props file has to be imported during versioning.targets
+    in order to get all the version properties set correctly.
+    So overwrite these two properties here, so they are set correctly.
+    -->
+    <Company>Microsoft Corporation</Company>
```

Comment:
We may need a way to override these at some point.

Reply:
Do you want me to add a property that can disable overwriting these properties?

My thought was, if you aren't a Microsoft .NET Framework assembly, you probably shouldn't be using buildtools.

Reply:
You can override them with this change.  You just need to do it after the buildtools versioning.targets file is imported.

Reply:
I'm fine with what we have for now. I can potentially imagine scenarios were we want a mix of at least the product name for different assemblies.